tests/components/application-connector/internal/testkit/test-api/oauth.go

```diff
@@ -50,12 +63,23 @@ func (oh *OAuthHandler) Token(w http.ResponseWriter, r *http.Request) {
 	}
 
 	token := uuid.New().String()
+	ttl := defaultTokenTTL
+
+	if ttlStr := r.URL.Query().Get(tokenLifetime); ttlStr != "" {
+		ttlOrErr, err := time.ParseDuration(ttlStr)
```

Comment:
Maybe `parsedTTL`? I assume it's not a TTL or error :D

Reply:
It's a TTL or not, I don't know what the idiomatic name should be, `parsedTTL` sounds good

Reply:
`exp` might be more approperiate, considering _expires in_ is the oauth standard, not _time to live_